docs/Targets.md

```diff
@@ -1609,6 +1609,8 @@ Identify the correct JLink device for your PIC32CK. In the examples the model is
    q
    ```
 
+3. Disconnect USB debugger and power cycle board. LED0 will illuminate (indicating version 1). Hit reset button and LED1 will illumate (indicating version 2).
```

Comment:
typo 'illuminate' (second occurrence). Kinda sounds convoluted tho? Can we just say it turns on?